nms/app/packages/magmalte/app/views/equipment/FEGGatewayDetailMain.js

```diff
@@ -149,6 +159,17 @@ function FEGGatewayOverview() {
               />
               <FEGGatewayDetailStatus refresh={refresh} />
             </Grid>
+            <Grid item>
+              <CardTitleRow
+                icon={PeopleIcon}
+                label="Managed Subscribers"
+                filter={() => filter(refreshSubscribers, setRefreshSubscribers)}
+              />
```

Comment:
Could you add a tooltip here, actually? Since we've had to clarify the definition of "Managed Subscribers", users will likely need the clarification too.